cloud_testing/kotlin_poc/flank.yml

```diff
@@ -12,3 +12,11 @@ limitBreak: false
 waitForResults: true
 testMethods:
   - com.example.app.ExampleUiTest#testPasses
+devices:
+  - model: Nexus6
```

Comment:
I think the default here should be NexusLowRes. Then in the test fixture yaml, we can use the real devices.